platform/src/main/java/org/hillview/remoting/ClusterDescription.java

```diff
@@ -0,0 +1,45 @@
+package org.hillview.remoting;
+
+import com.google.common.net.HostAndPort;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import org.hillview.dataset.api.IJson;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+/**
+ * Describes the list of hosts that comprise a cluster
+ */
+public final class ClusterDescription implements IJson {
+    private final List<HostAndPort> serverList;
+
+    public ClusterDescription(final List<HostAndPort> serverList) {
+        this.serverList = serverList;
+    }
+
+    public List<HostAndPort> getServerList() {
+        return this.serverList;
+    }
+
+
```

Comment:
Maybe you can add a comment showing an example little json representation of a ClusterDescription.